Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Data.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Data.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private List<Customer> _customerList = new();
+
+        public void Create(Customer customer)
+        {
+            var checkingCustomerCpf = _customerList.Any(customerElement => customerElement.Cpf == customer.Cpf);
+
+            var checkingCustomerEmail = _customerList.Any(customerElement => customerElement.Email == customer.Email);
+
+            if (checkingCustomerCpf == true) throw new ArgumentException($"Customer for Cpf: {customer.Cpf} already exists!");
+
+            if (checkingCustomerEmail == true) throw new ArgumentException($"Customer for Email: {customer.Email} already exists!");
```

Comment:
1. Se uma validação dar erro podemos estourar exceção e parar o fluxo ai mesmo, não devemos esperar as 2 validações terminarem 
2. Não precisa validar se um boolean é `true ou false`, basta fazer `if (seuBool)` para verificar true e `if (!seuBool)` para verificar false
CC: @gregoryzimmerwarren 

```suggestion
            var cpfAlreadyExists = _customerList.Any(customerElement => customerElement.Cpf == customer.Cpf);
            if (cpfAlreadyExists) throw new ArgumentException($"Customer for Cpf: {customer.Cpf} already exists!");
            
            var emailAlreadyExists = _customerList.Any(customerElement => customerElement.Email == customer.Email);
            if (emailAlreadyExists) throw new ArgumentException($"Customer for Email: {customer.Email} already exists!");
```